packages/compass-import-export/src/modules/import.ts

```diff
@@ -22,7 +22,7 @@
  * action or not. Not great, but it has saved me a considerable amount of time vs.
  * larger scale refactoring/frameworks.
  */
-
+import _ from 'lodash';
```

Comment:
This whole file is going to be a tricky diff to view because of the feature flag and branch.